client/src/store/musicPlayerSlice.ts

```diff
@@ -107,6 +108,15 @@ export const fetchRepeat = createAsyncThunk(
   },
 );
 
+export const fetchVolume = createAsyncThunk(
+  'musicPlayer/fetchVolume',
+  async (curVol: string) => {
+    const volAsNum = parseInt(curVol, 10);
```

Comment:
Just side note you can use the "+" in from of your curVol property to convert it to an integer